lib/compat/wordpress-6.2/class-gutenberg-rest-templates-controller-6-2.php

```diff
@@ -0,0 +1,77 @@
+<?php
+/**
+ * REST API: Gutenberg_REST_Templates_Controller_6_2 class
+ *
+ * @package    Gutenberg
+ * @subpackage REST_API
+ */
+
+/**
+ * Base Templates REST API Controller.
+ */
+class Gutenberg_REST_Templates_Controller_6_2 extends Gutenberg_REST_Templates_Controller {
+
+	/**
+	 * Registers the controllers routes.
+	 *
+	 * @return void
+	 */
+	public function register_routes() {
+
+		register_rest_route(
+			$this->namespace,
+			'/' . $this->rest_base . '/lookup',
```

Comment:
why lookup? Should it be `/fallback`?

Reply:
Oh this doesn't look like a new endpoint, what changed in this PR then?

Reply:
My changes are in a separate commit at https://github.com/WordPress/gutenberg/pull/46248/commits/505d0c0e7fafbfe87d796f9d58dd3825f7102f6c. The rest is just moving the endpoint.